### ISSUE TITLE
Ability to customize the diff colors

### ISSUE THREAD

**Issue by tunttunen** [ForkIssues/TrackerWin#1504]
In dark mode, the red and green are "way too close to each other" especially when concerning red green color blindness. In light mode the colors are fine, but in dark mode I feel that the colors require more brightness and/or saturation to pop out bit more. Would be awesome though to have customizable colors for the diff.

**Comment by DanPristupov** [comment]
Currently there's no way to customize them.

You can find useful the 'Show +- marks' option in the Preferences.

**Comment by flawesomesoft** [comment]
I've the exact same issue. I was creating an issue for that too but then i've noticed yours...

The option to show "-+" marks is a good workaround but not as convenient.

In general a custom color configuration would be very helpful.
Not only for the diff view but for example also for the graphs (as mentioned here #1328)

**Comment by ParamjotSingh5** [comment]
I also experienced this, in the context of a dark theme, colors seem almost identical. It would be good if we could switch these color combinations when the dark theme is enabled.

P.S.: I have been using a `fork` for a long time. I loved its UI and the branch graph. If in case, `fork` is open-source, I would love to contribute. 
Good day :)

**Comment by bifacil** [comment]
Yes please. Colors in light mode are also difficult to see for some people under specific lightning conditions...

Please, allow colors customizations...

**Comment by rarepops** [comment]
I was actually looking at a colleague over some code, but they are colorblind and I looked in the options for a colorblind option, without finding anything.

@DanPristupov can you please add this as a goal? Just a simple checkbox for colorblind mode should do (ideally a drop-down with the top 3 most common colorblindness) should also be enough
